src/metaprob/builtin_impl.clj

```diff
@@ -49,7 +49,7 @@
 
 ;; In metaprob, these are strict functions.
 
-(defn neq [x y] (not (= x y)))
+(def neq not=)
```

Comment:
Not as skeptical of this new thing as I am of `nil?`. Same reasoning applies but I think I have higher prior trust that `not=`, which I've never heard of, is the negation of `=`. Don't know why.